### PR TITLE
Contact form section

### PR DESCRIPTION
**Pull request by ernestkorir**
Add the Contact section and has forms for the name, email and message

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,10 +169,26 @@ <h5 class="media-text">LET’S CONNECT</h5>
...
         </div>
 
     </section>
+    <section class="contact-section">
+        <h2 class="contact-heading">Contact me</h2>
+        <p class="contact-cta">If you have an application you are
+            interested in developing, a feature
+            that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form action="#" class="contact-form">
```
Comment (leonardodiasb) [inline index.html]: - [ ] Your form is missing the action and method attribute to integrate with [formspree service](https://formspree.io/html). Kindly follow the 'get started' section and make sure you have a formspree action link so the form can be submitted.

File: index.html [hunk 2]
```diff
@@ -169,10 +169,26 @@ <h5 class="media-text">LET’S CONNECT</h5>
...
+            <input type="text" id="name" placeholder="Yeremaias" maxlength="30" />
+            <input type="email" placeholder="natajayanj14@gmail.com" id="email" />
+            <textarea name="message" id="message" cols="25" rows="10" placeholder="Write your message here"></textarea>
```
Comment (leonardodiasb) [inline index.html]: - [ ] The project requirements mention that the user cannot submit the form if any of the inputs are missing. Please make sure that all the inputs are required.
 - [ ] The message should have a limit of 500 characters. Please make sure that there is a limit of characters for this input as you did for the name input.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @ernestkorir, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - CSS style matching with Figma template. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Branch should have a name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section'). It is not clear what is this branch about by looking at its name. Kindly make sure you chose a better name for the next projects.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -169,10 +169,27 @@ <h5 class="media-text">LET’S CONNECT</h5>
...
+        <p class="contact-cta">If you have an application you are
+            interested in developing, a feature
+            that you need built or a project that
+            needs coding. I’d love to help with it </p>
+        <form action="https://formspree.io/f/{form_id}" class="contact-form" method="post">
+            <input type="text" id="name" placeholder="Yeremaias" maxlength="30" required />
+            <input type="email" placeholder="natajayanj14@gmail.com" id="email" required />
+            <textarea name="message" id="message" cols="25" rows="10" maxlength="500"
+                placeholder="Write your message here" required></textarea>
+            <button id="submit" type="submit">Get in Touch</button>
```
Comment (1GORDON) [inline index.html:188]: Great job team 🤝

- [ ] [Optional] Kindly implement the button interactions of `hover`, `enable` and `active` on this button.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @ernestkorir & @akos29 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**